flake.nix

```diff
@@ -13,6 +13,17 @@
 {
   description = "Development shell for Tide Disco";
 
+  nixConfig = {
+    extra-substituters = [
+      "https://espresso-systems-private.cachix.org"
```

Comment:
This really shouldn't be in a public repository...

Reply:
As discussed offline it is safe to put these in a public repo, but not ideal because people outside the organization can't use the cache, and their builds are much slower. Made [an issue](https://github.com/orgs/EspressoSystems/projects/20/views/2?pane=issue&itemId=44616921) to improve this later